specification/ml/evaluate_data_frame/MlEvaluateDataFrameRequest.ts

```diff
@@ -37,16 +37,15 @@ export interface Request extends RequestBase {
   body: {
     /**
      * Defines the type of evaluation you want to perform.
-     * @doc_url https://www.elastic.co/guide/en/elasticsearch/reference/current/evaluate-dfanalytics.html#ml-evaluate-dfanalytics-resources
+     * See [Data frame analytics evaluation resources](https://www.elastic.co/guide/en/elasticsearch/reference/current/evaluate-dfanalytics.html#ml-evaluate-dfanalytics-resources).
      */
     evaluation: DataframeEvaluationContainer
     /**
-     * Defines the index in which the evaluation will be performed.
+     * Defines the `index` in which the evaluation will be performed.
      */
     index: IndexName
     /**
-     * A query clause that retrieves a subset of data from the source index.
-     * @doc_url https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html
+     * A query clause that retrieves a subset of data from the source index. See [QueryDSL](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html).
```

Comment:
@lcawl do you prefer the link as markdown link or with `@doc_url`?
If provided with `@doc_url`, it will be easier to access in the resulting JSON, while if added in the description you can't easily access it if needed.

Reply:
My preference is [doc_id](https://github.com/elastic/elasticsearch-specification/blob/main/docs/modeling-guide.md#doc_id), so we don't have to clean up these URLs in multiple places when we move to the new docs system. Is there somewhere we can start recording those ID<->URL mappings?

Reply:
Unfortunately we don’t know how to find the appropriate doc_id, where can we find it?

Reply:
Could it just be a flat file that contains keywords and URLs, which is checked when we run make contrib (i.e. substitute the keywords for the URLs in the output)? The goal would be to just isolate all the URLs in one place, so they're easier to maintain. The way they accomplished this in Kibana is per this file, but we could probably be even simpler in this case: https://github.com/elastic/kibana/blob/main/src/core/public/doc_links/doc_links_service.ts